### ISSUE TITLE
Incorporate feedback from accessibility audit

### ISSUE THREAD

**Issue by iNPUTmice** [iNPUTmice/Conversations#3851]
We expect an accessibility audit of Conversations to be performed around August 20th. The feedback from which should be incorporated into the app.

**Comment by iNPUTmice** [comment]
Most parts from the accessibility review have been incorporated into 3c0773c6e71398a0e4de5440b009cfeedd758b85...8d64e101c98a9f33f54db1c7ea41c3f1f0b15f4b
